app/index.ts

```diff
@@ -55,25 +55,98 @@ function createWindow() {
 //
 // Create the window when the application is ready.
 //
-app.on("ready", createWindow);
+app.on("ready", () => {
+    createWindow();
+});
 
 
 //
 // Terminates the application when all windows have been closed.
 //
 app.on("window-all-closed", () => {
-    if (process.platform !== "darwin") {
-        app.quit();
-    }
+    //if (process.platform !== "darwin") {
+    //    app.quit();
+    //}
```

Comment:
Just kill this, no need to leave the comment lying around. It's in the revision history.